com.github.hakandundar34coding.mini-system-monitor.yml

```diff
@@ -0,0 +1,106 @@
+app-id: io.github.hakandundar34coding.mini-system-monitor
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: mini-system-monitor
+finish-args:
+  - --share=ipc
+  # Required for monitoring data download/upload speed.
+  - --share=network
+  - --socket=x11
+  # Required for monitoring data download/upload speed.
+  - --device=all
+  # Required for saving application configuration file.
```

Comment:
no it is not.

Reply:
It can not detect devices if this is disabled from Flatseal.

Reply:
xdg-config? I doubt.

Reply:
It was for ```- --device=all``` or ```- --share=network```.
xdg-config is used for saving settings into user config directory. XDG_CONFIG_HOME is not defined on some distributions.

Reply:
it is not necessary.